rafs/src/metadata/direct_v6.rs

```diff
@@ -33,6 +33,7 @@ use std::slice;
 use std::sync::Arc;
 
 use arc_swap::{ArcSwap, Guard};
+use nydus_utils::metrics::Inode;
```

Comment:
This change seems wrong, we should use Inode from crate::metadata::Inode

Reply:
Sorry, I fixed it already.